cluster-autoscaler/cloudprovider/azure/azure_scale_set_instance_cache.go

```diff
@@ -224,6 +225,8 @@ func (scaleSet *ScaleSet) instanceStatusFromVM(vm *compute.VirtualMachineScaleSe
 	case string(compute.GalleryProvisioningStateCreating):
```

Comment:
Let's also cleanup these "Gallery*" constants at some point; even if there is no correct constant in compute (I don't think there is), let's have something reasonably named within the provider